flytekit/image_spec/default_builder.py

```diff
@@ -405,6 +405,15 @@ def create_docker_context(image_spec: ImageSpec, tmp_dir: Path):
     else:
         extra_copy_cmds = ""
 
+    uv_image = "ghcr.io/astral-sh/uv:0.5.1"
+    micromamba_image = "mambaorg/micromamba:2.0.3-debian12-slim"
+    if image_spec.builder_config is not None:
+        if image_spec.builder_config.get("uv_image") is not None:
+            uv_image = image_spec.builder_config.get("uv_image")
+
+        if image_spec.builder_config.get("micromamba_image") is not None:
+            micromamba_image = image_spec.builder_config.get("micromamba_image")
```

Comment:
Minor nit:

```suggestion
    if image_spec.builder_config is not None:
        uv_image = image_spec.builder_config.get("uv_image", DEFAULT_UV_IMAGE)
        micromamba_image = image_spec.builder_config.get("micromamba_image", DEFAULT_MICROMAMBA_IMAGE)
```

where the defaults are constants at the module level:

```python
DEFAULT_UV_IMAGE = "ghcr.io/astral-sh/uv:0.5.1"
DEFAULT_MICROMAMBA_IMAGE = "mambaorg/micromamba:2.0.3-debian12-slim"
```

Reply:
Thanks for the advice! It is more readable.